pkg/tfbridge/schema.go

```diff
@@ -1407,7 +1406,12 @@ func extractInputs(oldInput, newState resource.PropertyValue, tfs shim.Schema, p
 }
 
 func getDefaultValue(tfs shim.Schema, ps *SchemaInfo) interface{} {
-	if dv, _ := tfs.DefaultValue(); dv != nil {
+	if dv, err := tfs.DefaultValue(); dv != nil {
+		if err != nil {
+			// Log error output but continue otherwise.
+			// This avoids a panic on preview such as https://github.com/pulumi/pulumi-cloudflare/issues/460.
+			glog.V(9).Infof(err.Error())
```

Comment:
If a user default value function returns an error, I would expect us to halt the `pulumi up` and display the error to the user. Swallowing a generic error, especially on v9, seems dangerous.

Reply:
The issue here is that there’s no fix for the user. We need to implement proper default value handling in the plug-in framework as a followup. Adding this error but emitting it as Info I felt was more informative than simply returning nil.

Reply:
Nope, we can't halt pulumi up in this case! This is the entire point of this PR. The code path is used to try to make import generated code better by taking default values into account, but Plugin Framework does not at this point support detecting default values, so the code must proceed with best-effort like returning nil. I think the behavior as written is what we want (modulo code style).